packages/neutrino-preset-jest/src/index.js

```diff
@@ -94,8 +94,12 @@ module.exports = (neutrino, opts = {}) => {
     return new Promise((resolve, reject) => {
       const configFile = join(tmpdir(), 'config.json');
       const options = normalizeJestOptions(opts, neutrino, args);
-      const cliOptions = { config: configFile, coverage: args.coverage, watch: args.watch };
+      const cliOptions = merge(
+        omit(['inspect', 'quiet', '_', 'version', 'help', 'use', 'options', '$0', 'files'], args),
+        { config: configFile, coverage: args.coverage, watch: args.watch }
+      );
 
+      cliOptions.findRelatedFiles = cliOptions.findRelatedFiles.split(' ');
```

Comment:
I'm not sure I understand this line. The command lint-staged is calling `jest --findRelatedTests file1.js file2.js file3.js`. Is this aligned with this `split` behaviour?